src/qilib/utils/storage/interface.py

```diff
@@ -100,6 +100,21 @@ def load_data(self, tag: TagType) -> Any:
             NoDataAtKeyException: if there is no data for the specified tag.
         """
 
+    @abstractmethod
+    def load_individual_data(self, property_name: Any, tag: TagType) -> Any:
```

Comment:
I think that property isn't the right name. This is becasue we talk about system properties, but for qilib these routines can handle all kinds of data. I think (you have to check mongo doc) that these are called fields in mongo-terms.

Reply:
Updated the property_name to field